src/components/ListItem.jsx

```diff
@@ -1,5 +1,39 @@
 import './ListItem.css';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({
+	name,
+	itemId,
+	listId,
+	dateLastPurchased,
+	totalPurchases,
+	updateItem,
+}) {
```

Comment:
ListItem gets 7 props, 4 of which are properties of item. Would passing the whole item object as a prop work in this situation? Is there a good reason not to? Genuinely curious, not asking for a change here.

Reply:
Oh! I like that idea. I think that it would simplify things quite a bit b/c then we could have `updateItem` just take `item` too! :-)

Reply:
I think this a good Idea and we updated the code to include these changes.